### ISSUE TITLE
Mew cannot learn TM73 (Thunder Wave)

### ISSUE THREAD

**Issue by Keyacom** [BuffelSaft/pokeemerald#325]
Mew is supposed to learn all TM moves (including Freeze-Dry), but somehow, despite learning the move in the official games, Mew cannot learn Thunder Wave in InclEm __at all__.

**Comment by BuffelSaft** [comment]
Yeah, this has already been reported and fixed in 2.0. It'll be tricky to track bugs until 2.0 is released, seeing as they get re-reported if I clear them from the bugs doc and I end up re-checking them if I don't clear them. 

Anyway, nice to see a github issue from you Keyacom! I appreciate all your typo/outdated move data bug reports, they'll eventually get fixed after the game stops exploding because someone used bug bite.
